src/upgradeConfig.js

```diff
@@ -94,18 +100,21 @@ function changePlugins(config) {
       }
     }
   }
+  if (config.plugins) {
+    config.plugins = config.plugins.concat(...updatedPlugins);
+  }
 }
 
 module.exports = function upgradeConfig(config, options) {
   config = Object.assign({}, config);
 
-  changePresets(config, options);
-  changePlugins(config);
+  let updatedPlugins = changePresets(config, options);
+  changePlugins(config, updatedPlugins);
 
   if (config.env) {
     Object.keys(config.env).forEach((env) => {
-      changePresets(config.env[env]);
-      changePlugins(config.env[env]);
+      let updatedPlugins = changePresets(config.env[env]);
```

Comment:
`newPlugins` instead of `updatedPlugins`, since we didn't update the plugins list yet?